index.html

```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>MyPortfolio</title>
+    <link rel="stylesheet" href="style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Inter:wght@400;500;600;700&display=swap">
+    <script src="https://kit.fontawesome.com/366c518b05.js" crossorigin="anonymous"></script>
+</head>
+<body>
+    <header>
+        <nav>
+            <a href="#" class="firstTitle">Welcome</a>
+            <a href="#" class="hamburgerButton"><i class="fa-solid fa-bars"></i></a>
+        </nav>
+    </header>
+    <section class="heroIntro">
+        <div class="heroIntroContainer">
+            <h1 class="heroIntroHeading">
+                Hey there. I'm<br>Troos<span class="spanText"><br>I'm a software<br>developer</span>
+                <br>
+                <br>
+            </h1>
```

Comment:
You're doing great 👍🏽 but here you're putting two things inside of one, you're putting the title and the paragraph inside of the `h1`, make sure to remove the paragraph inside of this and apply the requirements below 👇🏽 

- [ ] The paragraph (e.g., "I'm a software ...") should be a `p` tag.

   * This way will be semantically correct and the screen readers and other technologies will thank you